dask_sql/utils.py

```diff
@@ -48,6 +48,13 @@ def convert_to_datetime(df):
     return df
 
 
+def is_cudf(obj):
+    """
+    Check if an object is a cuDF type
+    """
+    return "cudf" in str(obj)
```

Comment:
Was thinking something like

```suggestion
    return "cudf" in (str(type(obj)), str(getattr(obj, "_partition_type", ""))
```